src/routes/userRoutes.ts

```diff
@@ -14,7 +14,7 @@ const router: Router = express.Router();
 
 router.get('/', UserController.getUsers);
 router.post('/registerUser', UserController.registerUser);
-router.get('/isVerified/:token', UserController.isVerified);
+router.post('/isVerified/:token', UserController.isVerified);
```

Comment:
Someone might have already integrated the endpoint, so you will need to tell them that you have changed the method.